sql/core/src/main/scala/org/apache/spark/sql/internal/HiveSerDe.scala

```diff
@@ -88,7 +89,16 @@ object HiveSerDe {
   }
 
   def getDefaultStorage(conf: SQLConf): CatalogStorageFormat = {
-    val defaultStorageType = conf.getConfString("hive.default.fileformat", "textfile")
+    // To respect hive-site.xml, it peeks Hadoop configuration from existing Spark session,
```

Comment:
Small correction is required in comments
Nit: it picks

Reply:
oh, I thought `peeks` makes sense too. It takes a look for Hadoop configuration :).

Reply:
yeah you are right, it makes sense too :)